{"uid":"ffc847e659ce146b","name":"test_backup_gcp","fullName":"tests.integration.test_backups#test_backup_gcp","historyId":"47bf3e993ce19fe3a2c12f6f7294fe65","time":{"start":1731808775490,"stop":1731810105453,"duration":1329963},"description":"Build and deploy two units of PostgreSQL in GCP and then test the backup and restore actions.","descriptionHtml":"<p>Build and deploy two units of PostgreSQL in GCP and then test the backup and restore actions.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/test_backups.py\", line 215, in test_backup_gcp\n    await backup_operations(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/helpers.py\", line 858, in backup_operations\n    await ops_test.model.wait_for_idle(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1731808775098,"stop":1731808775098,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1731808775098,"stop":1731808775098,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"github_secrets","time":{"start":1731808775098,"stop":1731808775098,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1731808775098,"stop":1731808775422,"duration":324},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_configs","time":{"start":1731808775422,"stop":1731808775489,"duration":67},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1731808775490,"stop":1731808775490,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy two units of PostgreSQL in GCP and then test the backup and restore actions.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/test_backups.py\", line 215, in test_backup_gcp\n    await backup_operations(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/helpers.py\", line 858, in backup_operations\n    await ops_test.model.wait_for_idle(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","steps":[],"attachments":[{"uid":"7ea2c160870d1886","name":"log","source":"7ea2c160870d1886.txt","type":"text/plain","size":7647}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1731810105457,"stop":1731810105457,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_configs::finalizer","time":{"start":1731810105465,"stop":1731810106235,"duration":770},"status":"broken","statusMessage":"botocore.exceptions.NoCredentialsError: Unable to locate credentials\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/test_backups.py\", line 100, in cloud_configs\n    for bucket_object in bucket.objects.filter(Prefix=config[\"path\"].lstrip(\"/\")):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 79, in __iter__\n    for page in self.pages():\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 169, in pages\n    for page in pages:\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 269, in __iter__\n    response = self._make_request(current_kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 357, in _make_request\n    return self._method(**current_kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 569, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1005, in _make_api_call\n    http, parsed_response = self._make_request(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1029, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 119, in make_request\n    return self._send_request(request_dict, operation_model)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 196, in _send_request\n    request = self.create_request(request_dict, operation_model)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 132, in create_request\n    self._event_emitter.emit(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 412, in emit\n    return self._emitter.emit(aliased_event_name, **kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 256, in emit\n    return self._emit(event_name, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 239, in _emit\n    response = handler(**kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 105, in handler\n    return self.sign(operation_name, request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 197, in sign\n    auth.add_auth(request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/botocore/auth.py\", line 423, in add_auth\n    raise NoCredentialsError()\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1731810106236,"stop":1731810106625,"duration":389},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1731810106626,"stop":1731810106626,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1731810106626,"stop":1731810106626,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1731810106626,"stop":1731810106626,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(2)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_backups"},{"name":"host","value":"github-runner"},{"name":"thread","value":"18418-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_backups"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ad67fd9fc40e2f31","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":119,"skipped":0,"passed":0,"unknown":3,"total":122},"items":[{"uid":"57b7f32241ba7b61","reportUrl":"../240//#testresult/57b7f32241ba7b61","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731725987061,"stop":1731727807647,"duration":1820586}},{"uid":"bacf9c0c2768a319","reportUrl":"../239//#testresult/bacf9c0c2768a319","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731637882675,"stop":1731639212416,"duration":1329741}},{"uid":"ee06d217ae6d8918","reportUrl":"../238//#testresult/ee06d217ae6d8918","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731551077295,"stop":1731552418208,"duration":1340913}},{"uid":"8a3a88f01ad54347","reportUrl":"../237//#testresult/8a3a88f01ad54347","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731461845264,"stop":1731463206996,"duration":1361732}},{"uid":"192f1768f6d13eb6","reportUrl":"../236//#testresult/192f1768f6d13eb6","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731379277586,"stop":1731380588653,"duration":1311067}},{"uid":"9e274ce7ddab7b72","reportUrl":"../235//#testresult/9e274ce7ddab7b72","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731294986449,"stop":1731296322306,"duration":1335857}},{"uid":"5dd397368785d0ce","reportUrl":"../234//#testresult/5dd397368785d0ce","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s-gcp/0 [idle] waiting: awaiting for cluster to start\n  postgresql-k8s-gcp/1 [idle] active: Primary","time":{"start":1731205151563,"stop":1731206184403,"duration":1032840}},{"uid":"ebda5a69cee8fa4","reportUrl":"../233//#testresult/ebda5a69cee8fa4","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1731121604121,"stop":1731122922240,"duration":1318119}},{"uid":"2798034504f355be","reportUrl":"../232//#testresult/2798034504f355be","status":"unknown","time":{}},{"uid":"585bc4632abdca77","reportUrl":"../231//#testresult/585bc4632abdca77","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730951285102,"stop":1730952614401,"duration":1329299}},{"uid":"41d9dec259bbf70b","reportUrl":"../230//#testresult/41d9dec259bbf70b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730859191118,"stop":1730860521039,"duration":1329921}},{"uid":"b1001638ab200e68","reportUrl":"../229//#testresult/b1001638ab200e68","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730772736529,"stop":1730774058345,"duration":1321816}},{"uid":"a2c797c317395a43","reportUrl":"../228//#testresult/a2c797c317395a43","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s-gcp/0 [executing] waiting: awaiting for member to start\n  postgresql-k8s-gcp/1 [idle] active: Primary","time":{"start":1730687659519,"stop":1730688680313,"duration":1020794}},{"uid":"cd6cb3f9c6627dca","reportUrl":"../227//#testresult/cd6cb3f9c6627dca","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s-gcp/0 [idle] active: Primary\n  postgresql-k8s-gcp/1 [idle] waiting: awaiting for member to start","time":{"start":1730598057399,"stop":1730599094286,"duration":1036887}},{"uid":"f818720e2c007c03","reportUrl":"../226//#testresult/f818720e2c007c03","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730514123925,"stop":1730515420228,"duration":1296303}},{"uid":"755d2e307b9ff22f","reportUrl":"../225//#testresult/755d2e307b9ff22f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730430568135,"stop":1730431901197,"duration":1333062}},{"uid":"625d692f8f19c0ce","reportUrl":"../224//#testresult/625d692f8f19c0ce","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730343552424,"stop":1730344845877,"duration":1293453}},{"uid":"7ed3a091bc7405e6","reportUrl":"../223//#testresult/7ed3a091bc7405e6","status":"unknown","time":{}},{"uid":"6ed67736be8035b8","reportUrl":"../222//#testresult/6ed67736be8035b8","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730169206723,"stop":1730170630270,"duration":1423547}},{"uid":"d5cae80a5245e88f","reportUrl":"../221//#testresult/d5cae80a5245e88f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ns3-integrator/0 [idle] blocked: Missing parameters: ['access-key', 'secret-key']","time":{"start":1730083018779,"stop":1730084411107,"duration":1392328}}]},"tags":["asyncio","abort_on_fail","group(2)"]},"source":"ffc847e659ce146b.json","parameterValues":[]}